docs: add release notes and artifacts for 20250707-1777929 (#6199)
influxdata/docs-v2: content/influxdb3/clustered/reference/release-notes/clustered.md

---

## 20250707-1777929 {date="2025-07-07"}

### Quickstart

```yaml
spec:
  package:
    image: us-docker.pkg.dev/influxdb2-artifacts/clustered/influxdb:20250707-1777929
```

#### Release artifacts

> - [app-instance-schema.json](/downloads/clustered-release-artifacts/20250618-1758428/app-instance-schema.json)
> - [example-customer.yml](/downloads/clustered-release-artifacts/20250618-1758428/example-customer.yml)

Reviewer: ```suggestion
- [app-instance-schema.json](/downloads/clustered-release-artifacts/20250707-1777929/app-instance-schema.json)
- [example-customer.yml](/downloads/clustered-release-artifacts/20250707-1777929/example-customer.yml)
```

- [InfluxDB Clustered README EULA July 2024.txt](/downloads/clustered-release-artifacts/InfluxDB%20Clustered%20README%20EULA%20July%202024.txt)

### Highlights

#### Database rename and undelete

> Databases can now be renamed and undeleted with [influxctl v2.10.2](https://docs.influxdata.com/influxdb3/clustered/reference/release-notes/influxctl/#v2102) or later.

Reviewer: ```suggestion
Databases can now be renamed and undeleted with [influxctl v2.10.2](/influxdb3/clustered/reference/release-notes/influxctl/#v2102) or later.
```

#### Table delete and list

> Tables can now be deleted and listed with [influxctl v2.10.2](https://docs.influxdata.com/influxdb3/clustered/reference/release-notes/influxctl/#v2102) or later.

Reviewer: ```suggestion
Tables can now be deleted and listed with [influxctl v2.10.2](/influxdb3/clustered/reference/release-notes/influxctl/#v2102) or later.
```

#### Faster ingester recovery

> The persist queue workers now scale up when the queue is saturated, attempting to more quickly get through the backlog and restore write availability.

Reviewer: ```suggestion
Persist queue workers now scale up when the queue is saturated to allow ingesters to get through the backlog quickly and restore write availability.
```
